CMakeLists.txt

```diff
@@ -348,6 +348,7 @@ add_library(mixxx-lib STATIC EXCLUDE_FROM_ALL
   src/library/baseplaylistfeature.cpp
   src/library/basesqltablemodel.cpp
   src/library/basetrackcache.cpp
+  src/library/basetrackcollectionfeature.cpp
```

Comment:
Please update build/depends.py until we remove scons

Reply:
ah, wasn't aware of that - the cmake update was done automatically by clion - will do